### PR TITLE
Task/zone units frontend

### PR DESCRIPTION
**Pull request by conbrad**
- Generalizes the concept of `FireZone` to `FireShape` throughout the codebase
- Adds migration for adding fire zone unit shape type
- Adds migration for calculating fire zone unit combustible area
- Points frontend layers to the fire zone unit pmtiles files
- Has SFMS data from September 1 to 7, 2023

# Test Links:
[Landing Page](https://wps-pr-3204.apps.silver.devops.gov.bc.ca/)
[MoreCast 2.0](https://wps-pr-3204.apps.silver.devops.gov.bc.ca/morecast-2)
[Percentile Calculator](https://wps-pr-3204.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3204.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3204.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3204.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3204.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3204.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3204.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/features/fba/components/map/Legend.tsx [hunk 1]
```diff
@@ -76,12 +76,12 @@ const LegendItem: React.FC<LegendItemProps> = ({ label, checked, onChange, subIt
...
   onToggleLayer: (layerName: string, isVisible: boolean) => void
   showZoneStatus: boolean
-  setShowZoneStatus: React.Dispatch<React.SetStateAction<boolean>>
+  setShowShapeStatus: React.Dispatch<React.SetStateAction<boolean>>
   showHFI: boolean
   setShowHFI: React.Dispatch<React.SetStateAction<boolean>>
 }
 
-const Legend = ({ onToggleLayer, showZoneStatus, setShowZoneStatus, showHFI, setShowHFI }: LegendProps) => {
+const Legend = ({ onToggleLayer, showZoneStatus, setShowShapeStatus, showHFI, setShowHFI }: LegendProps) => {
```
Comment (dgboss) [inline web/src/features/fba/components/map/Legend.tsx:84]: nit: Rename `showZoneStatus` to `showShapeStatus` for the sake of consistency.

File: web/src/features/fba/components/ZoneSummaryPanel.tsx [hunk 2]
```diff
@@ -29,10 +29,10 @@ const CentreName = styled(Typography)({
 })
 
 interface Props {
-  selectedFireZone: FireZone | undefined
+  selectedFireZone: FireShape | undefined
   fuelTypeInfo: Record<number, FireZoneThresholdFuelTypeArea[]>
   hfiElevationInfo: ElevationInfoByThreshold[]
-  fireZoneAreas: FireZoneArea[]
+  fireZoneAreas: FireShapeArea[]
```
Comment (dgboss) [inline web/src/features/fba/components/ZoneSummaryPanel.tsx:35]: nit: Rename to fireShapeAreas (take it or leave it!)

**Review by dgboss — APPROVED** [review]
Nice work! It looks great to me. Just one minor suggestion if you haven't already done it, consider checking in with Neal regarding the label choice for the zone unit polygons on the map. We're showing things like 'V8-North Island Mid Coast Fire Zone' as a label which seems a little awkward to me.

**Review by brettedw — APPROVED** [review]
Looks great! Agreed with Darren on the label question for Neal. 
One other labelling question I have is the label names between the lower mainland Fraser area and the Haida Gwaii area. I see they're separated advisory wise which is awesome, but wondering if Neal may want them labelled in a different way somehow. Maybe not since those names are probably technically correct

File: api/alembic/versions/d5115b761e39_compute_zone_unit_combustible_area.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
+"""compute zone unit combustible area
+
+Revision ID: d5115b761e39
+Revises: 5b745fe0bd7a
```
Comment (dgboss) [inline api/alembic/versions/d5115b761e39_compute_zone_unit_combustible_area.py:4]:
```suggestion
Revises: f2e027a47a3f
```

**Pull request merged**